src/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import 'bulma/css/bulma.css';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { tabs } from '../../data/tabs';
+
+export const Tabs = () => {
+  const { tabId } = useParams();
+  const selectedTabId = tabId || null;
+
+  const getContentById = (id: string | null) => {
+    const foundTodo = tabs.find(tab => tab.id === id);
+
+    return foundTodo ? foundTodo.content : 'Please select a tab';
+  };
+
+  const tabContent = getContentById(selectedTabId);
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={classNames({
+                'is-active': tab.id === tabId,
+              })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <Link to={`../${tab.id}`}>
```

Comment:
Relative path in the Link to is used. It's better to use absolute paths to prevent potential routing issues in case the structure of the routes changes. It should be '/tabs/${tab.id}'.